src/components/SelectListForm.jsx

```diff
@@ -1,7 +1,8 @@
 import { useNavigate } from 'react-router-dom';
 import { useState } from 'react';
+
 export default function SelectListForm({ data, listPath, setListPath }) {
-	const [selectedList, setSelectedList] = useState(listPath);
+	const [selectedList, setSelectedList] = useState('');
```

Comment:
Was there a specific reason you set initial selectedList state to ""? When you go from list to home (when a list has been selected), the dropdown no longer displays the list name.

Reply:
I was getting an error saying something along the lines of "initial listPath cannot be null, must be either undefined or an empty string"

Changing the useState there was the only way I could clear it, since doing a listPath = '' in the parameters wouldn't do the trick